src/main/java/dev/ikm/tinkar/sandbox/Sept2024ConnectathonStarterData.java

```diff
@@ -61,21 +63,20 @@ public Sept2024ConnectathonStarterData(File exportDataStore, File exportFile) {
                         TinkarTerm.PRIMORDIAL_MODULE,
                         TinkarTerm.PRIMORDIAL_PATH);
 
+        session = composer.open(State.ACTIVE, System.currentTimeMillis(),
```

Comment:
Should we use the open method that does not require a timestamp here?

Reply:
Yes! Good catch. Updated.